common/templates/context_funcs.go

```diff
@@ -31,20 +31,39 @@ func (c *Context) tmplSendDM(s ...interface{}) string {
 	c.GS.RUnlock()
 
 	info := fmt.Sprintf("Custom Command DM From the server **%s**", gName)
-
-	// Send embed
-	if embed, ok := s[0].(*discordgo.MessageEmbed); ok {
-		embed.Footer = &discordgo.MessageEmbedFooter{
-			Text: info,
+	msgSend := &discordgo.MessageSend{
+			AllowedMentions: discordgo.AllowedMentions{
+					Parse : []discordgo.AllowedMentionType{discordgo.AllowedMentionTypeUsers},
+			},
 		}
-
-		bot.SendDMEmbed(memberID, embed)
+	
+	switch t:= s[0].(type) {		
+		case *discordgo.MessageEmbed:
+			t.Footer = &discordgo.MessageEmbedFooter{
+					Text: info,
+				    }
+			msgSend.Embed = t
+		case *discordgo.MessageSend:
+			msgSend = t
+			if msgSend.Embed != nil {
+				msgSend.Embed.Footer = &discordgo.MessageEmbedFooter{
+					Text: info,
+				    }
+				break
+			}
+			if (strings.TrimSpace(msgSend.Content) == "") && (msgSend.File == nil) {
+				return ""
+			}
+			msgSend.Content = info + "\n" + msgSend.Content
+		default:
+			msgSend.Content = fmt.Sprintf("%s\n%s", info, fmt.Sprint(s...))
+	}
+	
+	channel, err := common.BotSession.UserChannelCreate(memberID)
```

Comment:
throw away the error here aswell as to not break existing cc's

Reply:
I am throwing out the error. I just avoid an extra endpoint call in case of an error and simply return "" early.

Reply:
ah you are, my bad